docs/Opportunities/Internal/TA.md

```diff
@@ -0,0 +1,32 @@
+---
+title: TA/Tutor
+tags:
+  - CAS
+  - Tandon
+  - Student
+sidebar_position: 5
+---
+
+# TA/Tutor
+
+Working as a Teaching Assistant (TA) or tutor presents a valuable and rewarding opportunity to engage with teaching on campus. By choosing an area that captivates you, you can support professors or aid fellow students in acquiring new knowledge, all the while reinforcing your own understanding of the material and receiving compensation. Additionally, these roles offer a chance to enhance your soft skills, including communication and presentation abilities. Explore the extensive opportunities available at CAS/Tandon and learn how to initiate your application process below.
+
+Grading and tutoring roles are generally more readily available for undergraduate students, whereas TA positions are mostly open to graduate students. You may expect a weekly workload of 5 to 20 hours, varying by course. International students should be mindful of their allowable working hour limits.
```

Comment:
For Tandon, it is common for undergraduate students to TA for courses as well; it is also course dependent whether a course has undergrad, grad, or both for TAs.